Lib/test/test_capi/test_long.py

```diff
@@ -451,11 +451,15 @@ def test_long_asnativebytes(self):
             (MAX_SSIZE, SZ),
             (MAX_USIZE, SZ + 1),
             (-MAX_SSIZE, SZ),
-            (-MAX_USIZE, SZ + 1),
+            (-MAX_USIZE, SZ),
```

Comment:
This change is incorrect. That value requires 129 bits to represent in twos compliment, thus SZ+1 bytes.  (same for the "# edge case" additions below)